docs/index.rst

```diff
@@ -92,22 +92,12 @@ We've divided up the documentation into four sections with different purposes:
 
             To the API reference
 
-.. toctree::
-    :hidden:
-    :caption: Tutorials
-
-    All Tutorials <tutorials/index>
-    tutorials/intro
-    tutorials/getting_started
-    Calibrations <tutorials/calibrations>
-    Data Processor <tutorials/data_processor>
-    Curve Analysis <tutorials/curve_analysis>
-    Visualization <tutorials/visualization>
-    Custom Experiments <tutorials/custom_experiment>
-
 .. toctree::
     :hidden:
 
+    Qiskit Experiments Home <self>
+    Getting Started <tutorials/getting_started>
```

Comment:
Adding `Getting Started` as a separate menu item to follow the recommended site structure: https://github.com/Qiskit/qiskit_sphinx_theme#tip-suggested-site-structure